blocks/card/card.css

```diff
@@ -0,0 +1,128 @@
+.card-box {
+  box-sizing: border-box;
+  border: 1px solid #dadada;
+  background-color: var(--color-background);
+  display: flex;
+  flex-direction: column;
+  border-radius: 1rem;
+  box-shadow: 0px 4px 8px 0px rgba(0, 0, 0, 0.1),
+    0px 1.4px 2.7px 0px rgba(0, 0, 0, 0.05), 0px 0px 1px 0px rgba(0, 0, 0, 0.16);
+}
+
+.card__body {
+  padding: 1.25rem 1rem;
+}
+
+.card__title {
+  color: var(--color-text-dark);
+  margin: 0;
+
+  @media (min-width: 48rem) {
+    font-size: var(--spectrum-title-size-m);
```

Comment:
Our `font-size`s for the card titles and card body text should use an `@container` query to make sure that the fonts scale according to the size of the card itself.